kitsune/flagit/tests/__init__.py

```diff
@@ -2,21 +2,15 @@
 
 from django.conf import settings
 from django.template.defaultfilters import slugify
+from django.test import override_settings
 
 from kitsune.sumo.tests import LocalizingClient, TestCase
 
 
+# Override the cache prefix with a test-specific one.
+@override_settings(
```

Comment:
You're right - this is a change in behaviour. I tried it and it worked so I didn't think anything of it. The correct way to do it if we do require a unique cache for each test would be to use the enter/exit methods of a `self.settings(...)` context manager. Your call - worth the change?

Reply:
Nah. If this works then great. I just found it interesting to point out in case issues arise in future. Maybe adding a comment to this affect would be nice if you get time and remember, but I'm not going to worry about it since it's working. Thanks!